trackier-ios-sdk/Classes/TrackierSDK.swift

```diff
@@ -71,4 +71,12 @@ public class TrackierSDK {
     public static func trackAsOrganic(organic: Bool) {
         shared.instance.organic = organic
     }
+    
+    public static func setUserPhone(userPhone: String){
+        shared.instance.customerPhone = userPhone
+    }
+    
+    public static func setUserName(userName: String){
```

Comment:
same as mentioned above
```suggestion
    public static func setUserName(userName: String) {
```